### PR TITLE
Add releaseCamera Method in Android

### PR DESCRIPTION
**Pull request by agrass**
To fix problem of multiples cameras running and crashing in Android devices, issue #263 

Example:
Before changing the route to prevent re render and the instance of another camera when come back, I do:

``` 
this.camera.releaseCamera({target: Camera.constants.CaptureTarget.disk});
```

### CODE REVIEW

File: Camera.js [hunk 1]
```diff
@@ -186,6 +186,22 @@ export default class Camera extends Component {
...
+      target: props.captureTarget,
+      quality: props.captureQuality,
+      type: props.type,
+      title: '',
+      description: '',
+      ...options
+    };
+    CameraManager.releaseCamera(options);
+  }
+
```
Comment (vespakoen) [inline Camera.js]: The releaseCamera method expects "type" to be in the options array, you pass in a lot of other stuff which seems unnecessary?

Comment (agrass) [inline Camera.js]: You are right, I'm going to remove the other options

**Comment by corbt** [comment]
I haven't tried this library with Android, but could the component be configured to automatically release the camera when it's unmounted? That seems more ideal.

**Comment by agrass** [comment]
You're right, @corbt . I closed this PR, because it's to fix something specific with `react-native-router-flux` or `NavigationExperimental`. I don't know if it's the best workaround but one fix in case of `react-native-router-flux` is unmounting the first camera before re open another one (using type='reset') when changing scene.

**Pull request closed without merging**